tensorflow/python/keras/activations.py

```diff
@@ -274,6 +298,8 @@ def sigmoid(x):
 
   >>> a = tf.constant([-20, -1.0, 0.0, 1.0, 20], dtype = tf.float32)
   >>> b = tf.keras.activations.sigmoid(a)
+  >>> b.numpy()
+  [2.0611537e-09, 2.6894143e-01, 5.0000000e-01, 7.3105860e-01, 1.0000000e+00]
   >>> b.numpy() >= 0.0
   array([ True,  True,  True,  True,  True])
```

Comment:
No need for the `.numpy()` line, also, let's remove the `>= 0.0` part of the docstring while we're here

Reply:
Please ignore the comments about the `.numpy()` calls

Reply:
Removed the >=0.0 line, but do we need to put a line saying it produces output in the range of 0 to 1

Reply:
Yes, please. Let's add that